src/resolver.js

```diff
@@ -103,12 +103,12 @@ function resolverFactory(target, options) {
     findOptions.context = context;
     findOptions.logging = findOptions.logging || context.logging;
 
-    findOptions = options.before(findOptions, args, context, {
+    findOptions = await Promise.resolve(options.before(findOptions, args, context, {
```

Comment:
Should also be applied here: https://github.com/mickhansen/graphql-sequelize/blob/master/src/generateIncludes.js#L75